### PR TITLE
fix datasets metadata fails with missing 'errorMessage'

### LINKED ISSUE #235: kaggle datasets metadata fails with KeyError: 'errorMessage' with API 1.5.6 and 1.5.5
```
kaggle datasets metadata -p ./ higepon/tf2-my-submission
```

```
Traceback (most recent call last):
  File "/opt/anaconda3/bin/kaggle", line 10, in <module>
    sys.exit(main())
  File "/opt/anaconda3/lib/python3.7/site-packages/kaggle/cli.py", line 51, in main
    out = args.func(**command_args)
  File "/opt/anaconda3/lib/python3.7/site-packages/kaggle/api/kaggle_api_extended.py", line 1028, in dataset_metadata_cli
    meta_file = self.dataset_metadata(dataset, path)
  File "/opt/anaconda3/lib/python3.7/site-packages/kaggle/api/kaggle_api_extended.py", line 1010, in dataset_metadata
    if (result['errorMessage']):
KeyError: 'errorMessage'
```

This is not happening in 1.5.4.

### PR DESCRIPTION
**Pull request by adrianlubitz**
This fixes #235

### CODE REVIEW

File: kaggle/api/kaggle_api_extended.py [hunk 1]
```diff
@@ -1044,10 +1044,12 @@ def dataset_metadata(self, dataset, path):
 
         result = self.process_response(
             self.metadata_get_with_http_info(owner_slug, dataset_slug))
-        if (result['errorMessage']):
+        print(result)
+        print(type(result))
```
Comment (Borda) [inline kaggle/api/kaggle_api_extended.py:1048]:
```suggestion
        print(result)
        print(type(result))

```
let's drop this debugging code...

**Comment by Borda** [comment]
@adrianlubitz lets user full PR name not use a number in title...

**Comment by adrianlubitz** [comment]
@Borda the number is the reference to the issue that this PR closes
#235 
I don't recall exactly but I think the title was auto-generated from the Issue

**Comment by Borda** [comment]
> @Borda the number is the reference to the issue that this PR closes #235 I don't recall exactly but I think the title was auto-generated from the Issue

I know so I would say that is much easier for anyone to keep track if the PR is named `fix datasets metadata fails with missing 'errorMessage` instead of `fix for #235` which does not say anything :rabbit:

**Pull request closed without merging**